Week1/homework/js-exercises/5-round-number.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+//Exercise 5: Round a number and log it
+
+//1-Declare a variable 'z' and assign the number 7.25 to it.
+const z = 7.25;
+
+//2-Write a 'console.log' statement in which you log the value of 'z'.
+console.log(z);
+
+//3-Declare another variable 'a' that has the value of 'z' but rounded to the nearest integer.
+const a = 7;
+
+//4-Write a 'console.log' statement in which you log the value of 'a'.
+console.log(a);
+
+//5-So now we have 'z' and 'a' find a way to compare the two values and store the highest of the two in a new variable.
+
+const highest = a > z ? a : z;

```

Comment:
Ah, nice! Do you know that you can also use [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max) ?